docs/configuration.md

```diff
@@ -64,5 +64,31 @@ the default retention for newly created keys that do not have a an override.
 #### Example
 
 ```
-$ redis-server --loadmodule ./redistimeseries.so 20
+$ redis-server --loadmodule ./redistimeseries.so RETENTION_POLICY 20
+```
+
+### DUPLICATE_POLICY
+
+Policy that will define handling of duplicate samples.
+The following are the possible policies:
+* BLOCK
```

Comment:
* `BLOCK` - an error will occur for any out of order sample
* `FIRST` - ignore the new value
* `LAST` - override with latest value
* `MIN` - only override if the value is lower than the existing value
* `MAX` - only override if the value is higher than the existing value

Reply:
let's duplicate this in the configuration and in the commands.  similar as a section to filtering https://oss.redislabs.com/redistimeseries/commands/#filtering